hooks/command

```diff
@@ -1,16 +1,53 @@
 #!/bin/bash
 set -euo pipefail
 
+set +u
+# Check if the plugin is disabled
+if [[ ! -z "$BUILDKITE_PLUGIN_GIT_DIFF_CONDITIONAL_DISABLE_PLUGIN" ]]; then
+  echo "Plugin disable flag detected, passing entire pipeline to buildkite"
+  buildkite-agent pipeline upload $BUILDKITE_PLUGIN_GIT_DIFF_CONDITIONAL_DYNAMIC_PIPELINE
+  exit 0
+fi
+set -u
+
+f_get_diff() {
+  local default_diff_commands
+  local diff_commands
+
+  default_diff_commands="git diff --name-only origin/master...HEAD,git diff --name-only HEAD HEAD~1"
+
+  IFS=','
+  read -a diff_commands <<< "${BUILDKITE_PLUGIN_GIT_DIFF_CONDITIONAL_DIFF:-$default_diff_commands}"
+
+  for diff_command in ${diff_commands[@]}; do
+    echo >&2 "Checking for diff using: ($diff_command)"
+
+    diff=$(eval $diff_command)
```

Comment:
SC2086: Double quote to prevent globbing and word splitting.
Try: diff=$(eval "$diff_command")